### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#477] [Esami & Esoneri M-Z]
### Traccia

<img width="712" alt="image" src="https://github.com/user-attachments/assets/1874f980-5ffd-4ef7-a69e-bb83ab1a45be">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/1d5d0e24-0d40-49c9-8fe9-dc402dcd2b10)
